changelogs/fragments/9019-onevnet-bugfix.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - one_vnet - fix ``VM_MAD`` to ``VN_MAD`` (https://github.com/ansible-collections/community.general/pull/9019).
+  - one_vnet - fix ``VMTEMPLATE`` to ``VNET`` (https://github.com/ansible-collections/community.general/pull/9019).
+  - one_vnet - fix loop for ``.AR_POOL.AR`` (https://github.com/ansible-collections/community.general/pull/9019).
+  - one_vnet - fix allocate parameter (https://github.com/ansible-collections/community.general/pull/9019).
```

Comment:
These changelog entries don't tell me anything about what happened to the module **from the point of view as a user**. Can you please rewrite them (and better reduce them to one fragment instead of four) so it's more clear what changes from a point of view of a user?

Reply:
My bad, I tried to be more verbose :)
Changed it in the last commit, could you please check it?

Reply:
Looks good to me!